File: Sources/11005/11005.java
/**
 *  BOJ 11005번 Java 소스 코드
 *  작성자 : 동동매니저 (DDManager)
 *
 *  ※ 실행 결과
 *  사용 메모리 : 14,304 KB / 540,672 KB
 *  소요 시간 : 112 ms / 3,000 ms
 *
 *  Copyright 2019. DDManager all rights reserved.
 */

import java.util.*;

public class Main{
	public static void main(String args[]){
		Scanner sc=new Scanner(System.in);
		int N=sc.nextInt();
		int B=sc.nextInt();
		System.out.println(Integer.toString(N,B).toUpperCase());
		sc.close();
	}
}

File: Sources/11XXX/11005/11005.kt
/**
 *  BOJ 11005번 Kotlin (Native) 소스 코드
 *  작성자 : 동동매니저 (DDManager)
 *
 *  ※ 실행 결과
 *  사용 메모리 : 14,248 KB / 278,528 KB
 *  소요 시간 : 4 ms / 1,000 ms
 *
 *  Copyright 2019. DDManager all rights reserved.
 */

fun main(args:Array<String>){
	var NB=readLine()!!.split(" ")
	var N=NB[0].toInt()
	var B=NB[1].toInt()
	println(N.toString(B).toUpperCase())
}